src/test/java/core/basesyntax/model/TransactionDtoTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.operationtypes.BalanceHandler;
+import core.basesyntax.service.operationtypes.OperationStrategy;
+import core.basesyntax.service.operationtypes.OperationStrategyImpl;
+import core.basesyntax.service.operationtypes.OperationTypeHandler;
+import core.basesyntax.service.operationtypes.Operations;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TransactionDtoTest {
```

Comment:
models, usually, not being tested. you can create a `mapper` that will be converting from a model to DTO or either way and test it